.github/workflows/backport.yml

```
@@ -0,0 +1,45 @@
+# Derived from https://github.com/NixOS/nixpkgs/blob/2ab6f6d61630889491f86396b27a76ffb6fbc7bb/.github/workflows/backport.yml
+
+name: Backport
+
+on:
+  pull_request_target:
+    types: [closed, labeled]
+
+permissions: {}
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+
+    if: >
+      (
+        github.repository_owner == 'danth' &&
+        github.event.pull_request.merged == true &&
+        (github.event_name != 'labeled' || startsWith('backport', github.event.label.name))
+      )
+
+    steps:
+      # Use a GitHub App rather than the default token so that GitHub Actions workflows
+      # may run on the created pull request
```

Consider wrapping at 80 charactes and adding a period at the end of the sentence:

```suggestion
      # Use a GitHub App rather than the default token so that GitHub Actions
      # workflows may run on the created pull request.
```